src/cascadia/TerminalSettingsEditor/ViewModelHelpers.h

```diff
@@ -72,3 +72,20 @@ public:                                                   \
 // setting, but which cannot be erased.
 #define PERMANENT_OBSERVABLE_PROJECTED_SETTING(target, name) \
     _BASE_OBSERVABLE_PROJECTED_SETTING(target, name)
+
+// Defines a basic observable property that uses the _NotifyChanges
+// system from ViewModelHelper.
+#define VIEW_MODEL_OBSERVABLE_PROPERTY(type, name, ...) \
```

Comment:
huh? this isn't different from WINRT_OBSERVABLE_PROPERTY

Reply:
`_propertyChangedHandlers` is private, so I can't use it as a param for WINRT_OBSERVABLE_PROPERTY.

`VIEW_MODEL_OBSERVABLE_PROPERTY` is the same as `WINRT_OBSERVABLE_PROPERTY` except it uses `_NotifyChanges` instead.

Reply:
Oh, because `_propertyChangedHandlers` is in the _parent_ class